controllers/containerimagebuild_controller.go

```diff
@@ -100,14 +100,21 @@ func (r *ContainerImageBuildReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		Timeout:        time.Duration(build.Spec.TimeoutSeconds) * time.Second,
 	}
 
+	log.Info(fmt.Sprintf("Building image %s and pushing to: %s", spec.ImageName, strings.Join(spec.PushRegistries, ", ")))
+	log.Info(strings.Repeat("=", 70))
```

Comment:
Is this delimiter important for fluentd/client view of the logs?

Reply:
Yep. I thought it might help, it's a little tough to distinguish the "push" messages from the buildkit logs (which are more verbose).

Reply:
word